lib/Optimizer/GraphOptimizer/GraphOptimizer.cpp

```diff
@@ -3014,6 +3043,9 @@ static void transformForPrecisionMode(const Backend &B, Function *F,
   if (precConfig.convertToFP16) {
     LOG_SCOPE(F->getLogContext(), "glow::convertFunctionToFloat16")
     convertFunctionToFloat16(F, precConfig);
+    FunctionPassManager FPM("FP16GraphOptzFPM",
```

Comment:
Instead of adding this, I think I'd prefer adding another `::glow::optimize(F, cctx, B)` after `transformForPrecisionMode()`/before the second `lower()` [here](https://github.com/pytorch/glow/blob/d2faa645149cbe32126fbdf91e2cbaaf137dae4d/lib/Optimizer/GraphOptimizer/GraphOptimizer.cpp#L3063). It's possible we could be missing out on other optimization opportunities for the lower-precision model prior to lowering.

Reply:
Yeah, actually I don't have a very good sense on all the optimization passes here. We can try.